src/core/backend/pearl/core.rs

```diff
@@ -215,8 +203,8 @@ impl BackendStorage for Pearl {
         }
     }
 
-    fn spawn_counter(&self) {
-        self.spawn_blobs_counter();
+    async fn blobs_count(&self) -> (usize, usize) {
+        Self::count_blobs(self.vdisks_groups.clone(), self.alien_vdisks_groups.clone()).await
```

Comment:
It seems like you can inline count_blobs here since it is the only place where it used